superset/charts/schemas.py

```diff
@@ -884,6 +884,9 @@ class AnnotationLayerSchema(Schema):
         allow_none=True,
     )
     show = fields.Boolean(description="Should the layer be shown", required=True)
+    showLabel = fields.Boolean(
+        description="Should the label always be shown", required=True,
```

Comment:
I am wondering whether we should make this optional to avoid a breaking change

Reply:
That's a good point. I think `allowNone=True` is better